src/gc/gcinterface.ee.h

```diff
@@ -128,6 +128,19 @@ class IGCToCLR {
     // barrier if it needs to be updated.
     virtual
     void StompWriteBarrier(WriteBarrierParameters* args) = 0;
+
+    // Signals to the finalizer thread that there are objects ready to
+    // be finalized.
+    virtual
+    void EnableFinalization() = 0;
+
+    // Returns true if there is additional work that the finalizer thread needs
+    // to do, outside of its duties of finalization.
+    virtual
+    bool HaveExtraWorkForFinalizer() = 0;
+
+    virtual
```

Comment:
I'm not familiar with this symbol I'm not sure of an appropriate comment for this method - does "g_fFinalizeRunOnShutDown" refer to finalizing objects during EE shutdown?